ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/forkchoice/ProposersDataManager.java

```diff
@@ -250,8 +249,8 @@ private Optional<PayloadBuildingAttributes> calculatePayloadBuildingAttributes(
             validatorRegistration,
             spec.getExpectedWithdrawals(state),
             currentHeadBlockRoot,
-            Optional.empty(),
-            Optional.empty()));
+            maxBlobsPerBlock.map(maxBlobs -> maxBlobs.dividedBy(2)),
+            maxBlobsPerBlock));
```

Comment:
We probably want to have a unit test for this part of the code on `ProposerDataManagerTest`.